### DISCUSSION TITLE
VSCode doesn't recognize git repo after opened with Script Kit

### DISCUSSION THREAD

**Discussion by austincrim-pfg** [johnlindquist/kit#163] [Q&A]
First of all, great project, John, I've loved tinkering with it so far!

I've been using an "Open folder in VSCode" script, inspired by Kent C Dodds', and I've noticed that Code doesn't recognize the project as containing a `.git` folder when opened through my script. I.e. if I click on the "SCM" tab of Code, it asks if I want to initialize a new repo, etc. I have tried opening folders with both ``exec(`code ${folder}`)`` and `edit(folder)`. 

Any thoughts?

Environment:
macOS: 10.15.7
VSCode: 1.55.2

**Comment by johnlindquist** [comment]
Did you happen to install git through homebrew or have any special git configuration?

I suspect we're not passing some required environment vars when launching code.

**Reply by austincrim** [reply]
I did install git through homebrew. Is there any specific git config I could provide or commands I could run to help debug?

**Reply by johnlindquist** [reply]
I might be able to fix it in the next build by passing along the entire PATH var.

I'm hoping to have a new beta build out tomorrow.

**Reply by austincrim** [reply]
Thanks, John. I'll keep an eye out for the next build and provide an update here if it solves my problem. I appreciate the responsiveness!

**Reply by johnlindquist** [reply]
@austincrim-pfg Would you try testing this?

```js
let PATH = [
  `/usr/local/bin`,
  `/usr/bin`,
  `/bin`,
  `/usr/sbin`,
  `/sbin`,
].join(":")

exec(`code ${folder}`, {
  env: {
    PATH,
  },
})
```

I think it might be the order in the PATH where code would use the default git install (instead of homebrew git).

**Reply by austincrim** [reply]
That works! I appreciate the help, John.

**Reply by johnlindquist** [reply]
Cool, thanks for reporting back!

I wonder if it's safe for me to switch that order by default 🤔. 
Or I can check if homebrew is installed, then switch 🤔 I'll probably do that
